src/Raven.Server/Documents/Queries/Dynamic/CollectionQueryRunner.cs

```diff
@@ -136,6 +136,10 @@ private async ValueTask ExecuteCollectionQueryAsync(QueryResultServerSide<Docume
 
                 var fieldsToFetch = new FieldsToFetch(query, null, IndexType.None);
                 var includeDocumentsCommand  = new IncludeDocumentsCommand(Database.DocumentsStorage, context.Documents, query.Metadata.Includes, fieldsToFetch.IsProjection);
+                
+                if (resultToFill is StreamQueryResult<Document> && resultToFill.IncludedPaths != null && resultToFill.IncludedPaths.Length > 0)
+                    throw new NotSupportedException("Includes are not supported by this type of query");
```

Comment:
Would prefer to mention that streaming is an issue

Reply:
Changed the message to `Includes are not supported by streaming query.`